lib/card.spec.ts

```diff
@@ -692,8 +696,8 @@ test('unlink will unlink all links between two cards with the specified verb', a
 	const cardSdk = new CardSdk(sdk as any);
 
 	await cardSdk.unlink(
-		account1 as any as core.Contract,
-		opportunity1 as any as core.Contract,
+		account1 as any as Contract,
+		opportunity1 as any as Contract,
```

Comment:
We should probably fix `unlink` method to accept something like this: `Pick<Contract, 'id' | 'type'>`, but not sure if it's under the scope of this pr.

Reply:
Ya this `as any as Contract` is not great. I'll take a closer look :+1:

Reply:
resolved by implementing your suggestion